src/App.js

```diff
@@ -73,6 +74,73 @@ function App() {
     setSearchText("");
   };
 
+  //filter state
+
+  const [withoutOxygen, setWithoutOxygen] = React.useState(false);
+  const [oxygen, setOxygen] = React.useState(false);
+  const [icu, setIcu] = React.useState(false);
+  const [icuwithventilator, setIcuWithVentilator] = React.useState(false);
+
+  const handleIcu = () => {
+    setIcu(!icu);
+    setOxygen(false);
+    setWithoutOxygen(false);
+    setIcuWithVentilator(false);
+  };
+
+  const handleOxygen = () => {
+    setOxygen(!oxygen);
+    setIcu(false);
+    setWithoutOxygen(false);
+    setIcuWithVentilator(false);
+  };
+
+  const handleIcuWithVentilator = () => {
+    setIcuWithVentilator(!icuwithventilator);
+    setOxygen(false);
+    setWithoutOxygen(false);
+    setIcu(false);
+  };
+
+  const handleWithoutOxygen = () => {
+    setWithoutOxygen(!withoutOxygen);
+    setOxygen(false);
+    setIcu(false);
+    setIcuWithVentilator(false);
+  };
+
```

Comment:
Then, here this can be simplified to one function,
```
	handleFilter(prop) {
		setFilterState({
			...defaultState, prop: !filterState[prop]
		})
	}
```